python/grass/jupyter/__init__.py

```diff
@@ -13,23 +13,64 @@
 #            License (>=v2). Read the file COPYING that comes with GRASS
 #            for details.
 
-"""Display classes and setup functions for running GRASS GIS in Jupyter Notebooks
+"""The *grass.jupyter* is a convenient GRASS GIS interface for Jupyter notebooks.
+
+Python is a great tool for data science and scientific computing. Jupyter_ is an
+environment with computational notebooks which makes it even better for tool for
+analysis and workflow prototyping. Computational notebooks are documents combining
+code, text, and results such as figures and tables. JupyterLab is an environment where
+you interact with all these parts. You can install it locally on your machine or
+use it online from some service provider.
 
 The *grass.jupyter* subpackage improves the integration of GRASS GIS and Jupyter
-Notebooks. The original version was written as part of Google Summer of Code in 2021
-and experimental version was included in GRASS GIS 8.0. Since then, much more
-development happened adding better session handling and rendering of additional data
-types.
+notebooks comparing to the standard Python API. The original version was written
+as part of Google Summer of Code in 2021 and experimental version was included in
+GRASS GIS 8.0. Since then, much more development happened adding better session
+handling and rendering of additional data types.
 
-For standard usage, simply import the top level package with a convenient alias, e.g.,::
+Usage
+=====
 
->>> import grass.jupyter as gj
+To start using it in a notebook, import the top level package with a convenient alias,
+such as *gj*, like this::
```

Comment:
is the double colon required here too?

Reply:
Yes. There is a code block afterwards. Are you thinking it is not needed for one line? In any case, this renders as:

![Screenshot from 2022-04-18 17-47-52](https://user-images.githubusercontent.com/5449060/163883089-a4bc669c-36c5-4fb4-a130-c9f09d34ad93.png)

The [double colons](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks) might be confusing, but reStructuredText [code-block](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block) directive lacks in simplicity. I don't know if enabling [Markdown in Sphinx](https://www.sphinx-doc.org/en/master/usage/markdown.html) for docstrings is an option (Doxygen can simply mix Markdown in, but that's not an option for Sphinx/Docutils/reStructuredText.